### ISSUE TITLE
[Filter] SIGSEGV when using EdgeTPU

### ISSUE THREAD

**Issue by IvanFilipov** [nnstreamer/nnstreamer#4433]
Hello, nnstreamer team!

Similar to #4194, 
I am also getting the error signal, when trying to run an edge tpu model 
with the corresponding "framework=edgetpu" parameter to tensor_filter.

My current setup is Rpi with Ubuntu 22.04, the latest NNStreamer from PPA and
Coral TPU USB accelerator.

What will be your suggestion to get it running? 
Is there another way except for the one mentioned by the author of the other bug - 
compiling nnstreamer with an older version of tflite?

I can see that you have some planned effort on the topic, perhaps we will have a stable fix soon?

All the best,
Ivan F.

**Comment by myungjoo** [comment]
You will need to use older tensorflow-lite for a while: https://github.com/nnstreamer/nnstreamer/issues/4194#issuecomment-1694036424

We do not have plans to solve this issue right now. We need volunteers or any team member with spare time to fix this.

**Comment by anyj0527** [comment]
@IvanFilipov You may give these [libedgetpu packages](https://github.com/feranick/libedgetpu/releases) a shot instead of NNStreamer PPA's ones. It is known as solving the libedgetpu segfault issue with recent tflite runtimes. If it does not work, our source code may need to be changed accordingly.

**Comment by IvanFilipov** [comment]
@anyj0527, thank you for your reply! 

I can confirm that using the _std_ version (https://github.com/feranick/libedgetpu/releases/download/16.0TF2.16.1-1/libedgetpu1-std_16.0tf2.16.1-1.ubuntu22.04_arm64.deb) has done the trick.

I have installed the lib first, then nnstreamer-edgetpu.

It works just fine on Rpi 4 with Ubuntu 22.04, but also
I have managed to run the setup in a docker container on a few different HW setups
(Coral dev board for example) and those tests were also successful.
